components/navigation.css

```diff
@@ -0,0 +1,24 @@
+.nav_items{
```

Comment:
use ( - )  instead of underscore in your class names

Reply:
done thank you.